services/deeplink/src/pages/buckets/share.tsx

```diff
@@ -19,7 +19,18 @@ const Margin = styled.div`
   height: 25px;
 `;
 
-const ShareBucket = ({ search }) => {
+interface ShareBucketProps {
```

Comment:
Does this need to be updated to work using the new sharing method (ie files instead of a bucket)?